content/docs/rfcs/17/README.md

```diff
@@ -184,12 +192,13 @@ The system parameters are summarized in the following table, and the recommended
 | `staked_fund` | the amount of ether to be staked by peers at the registration |
 | `reward_portion` | the percentage of `staked_fund` to be rewarded to the slashers |
 | `max_epoch_gap` | the maximum allowed gap between the `epoch` of a routing peer and the incoming message |
+| `acceptable_root_window_size` | The maximum number of past Merkle roots to store |
```

Comment:
Hmm. since proof is sent over the wire we *do* need to define the type strictly. However, for something like `acceptable_root_window_size`, I do not think we need to define the type, and leave it up to the implementer.